<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindowClass</class>
 <widget class="QMainWindow" name="mainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ANA Reservation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="myPushButton" name="BronzeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>180</width>
      <height>65</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>65</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="myPushButton" name="SFCButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>180</width>
      <height>65</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>65</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="myPushButton" name="PlatinumButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>180</width>
      <height>65</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>65</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="myPushButton" name="DiamondButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>180</width>
      <height>65</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>65</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>320</width>
      <height>289</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Tabiwari75DateLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>281</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1990/01/01</string>
    </property>
   </widget>
   <widget class="QLabel" name="PriorityDateLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>281</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1990/01/01</string>
    </property>
   </widget>
   <widget class="QLabel" name="NormalDateLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>281</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1990/01/01</string>
    </property>
   </widget>
   <widget class="QLabel" name="ReservationDateLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>予約開始日</string>
    </property>
   </widget>
   <widget class="QLabel" name="Tabiwari75Label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>111</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>旅割75</string>
    </property>
   </widget>
   <widget class="QLabel" name="PriorityLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>121</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2週間前</string>
    </property>
   </widget>
   <widget class="QLabel" name="NormalLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>111</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>通常</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myPushButton</class>
   <extends>QPushButton</extends>
   <header>mypushbutton.h</header>
   <slots>
    <slot>sl_SwitchIcon(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
